src/contexts/providers.context.tsx

```diff
@@ -140,6 +174,43 @@ const ProvidersProvider: FC = (props) => {
     }
   }, [connectedProvider]);
 
+  const estimateGasPrice = useCallback(
+    ({ chain, gasUnits }: EstimateGasPriceParams): Promise<BigNumber> => {
+      if (chain.provider === undefined) {
+        throw new Error("Provider is not available");
+      }
+
+      return chain.provider.getFeeData().then((feeData) => {
+        if (feeData.maxFeePerGas !== null && feeData.maxPriorityFeePerGas !== null) {
```

Comment:
Why check that `feeData.maxPriorityFeePerGas !== null` if we don't use `maxPriorityFeePerGas`?

Reply:
It's a leftover for a console.log hehe Fixed!